src/app/network/crypto.ts

```diff
@@ -139,51 +139,40 @@ export async function getEncryptedFile(
     const status = await readable.read();
 
     if (!status.done) {
-      hasher.process(status.value);
+      hasher.update(status.value);
       blobParts.push(status.value);
     }
 
     done = status.done;
   }
 
-  hasher.finish();
-
-  return [
-    new Blob(blobParts, { type: 'application/octet-stream' }),
-    createHash('ripemd160')
-      .update(Buffer.from(hasher.result as Uint8Array))
-      .digest('hex'),
-  ];
-}
-
-export function sha256(input: Buffer): Buffer {
-  return createHash('sha256').update(input).digest();
+  const ripemd160Hash = await ripemd160(hasher.digest());
```

Comment:
@sg-gs Why here we have `ripemd160` of `sha256`? I didn't change this part but I'm curious.

Reply:
Inherited from storj